authentik/policies/geoip/urls.py

```diff
@@ -0,0 +1,11 @@
+"""API URLs"""
+
+from django.urls import path
+
+from authentik.policies.geoip.api import GeoIPPolicyViewSet
+from authentik.policies.geoip.views import ISO3166View
+
+api_urlpatterns = [
+    ("policies/geoip", GeoIPPolicyViewSet),
+    path("iso-3166/", ISO3166View.as_view(), name="iso-3166-view"),
```

Comment:
There may or may not be a better place for this.

@BeryJu?

Reply:
I'd usually prefer to do it as `policies/geoip/iso3166/`, but obviously that could potentially cause errors due to that path also having the primary key...as a fallback I'd go with `policies/geoip_iso3166/` I guess?